tests/acceptance/bootstrap/AuthAppContext.php

```diff
@@ -154,4 +155,29 @@ public function theAdministratorCreatesAppTokenForUserWithExpirationTimeViaAuthA
 			)
 		);
 	}
+
+	/**
+	 * @When user :user deletes all the created auth-app tokens using the auth-app API
+	 *
+	 * @param string $user
+	 *
+	 * @return void
+	 */
+	public function userDeletesAllCreatedAuthAppTokenUsingAuthAppAPI(string $user): void {
+		$response = AuthAppHelper::listAllAppAuthTokensForUser(
+			$this->featureContext->getBaseUrl(),
+			$this->featureContext->getActualUsername($user),
+			$this->featureContext->getPasswordForUser($user),
```

Comment:
```suggestion
			$this->featureContext->getBaseUrl(),
			$this->featureContext->getActualUsername($user),
			$this->featureContext->getPasswordForUser($user),
```
these are repeated function calls
IMO, it will good to store in local variable and use it.